src/Signup.js

```diff
@@ -38,67 +39,109 @@ class Signup extends React.Component {
-      Zip: document.getElementById('adopterZip').value,
-    })
-    .then(() => {
-      console.log("Document successfully written!");
-      this.props.history.push("/home");
-    })
-    .catch((error) => {
-      console.error("Error writing document: ", error);
-    });
+    let boolin = this.signUp();
+    var user = firebase.auth().currentUser;
+    //console.log(boolin);
+    if (boolin) {
+      console.log(user.email);
+      var user = firebase.auth().currentUser;
+      document.getElementById("signup").style.display = "none";
+      // Modify Adopter document in collection "users"
+      if (user != null) {
+        db.collection("users").doc(user.email).set({
+          Name: document.getElementById('adopterName').value,
+          Age: document.getElementById('adopterAge').value,
+          Zip: document.getElementById('adopterZip').value,
+        })
+        .then(() => {
+          console.log("Document successfully written!");
+          this.props.history.push("/home");
+        })
+        .catch((error) => {
+          console.error("Error writing document: ", error);
+        });
+      }
+    }
+
+
   }
 
   render() {
     return (
-      <div className="signup" id="signup">
-        <div id="acc_type">
-          <h1 className="title">Account Type</h1>
-          <button onClick={this.adopter_acc} className="button b1 top">Adopter</button>
-          <button onClick={this.shelter_acc} className="button b1">Shelter</button>
-        </div>
+      <div className="big">
+        <div className="signup" id="signup">
+          <div id="acc_type">
+            <h1 className="title">Account Type</h1>
+            <br className="top" />
+            <button onClick={this.adopter_acc} className="button b1 top">Adopter</button>
+            <button onClick={this.shelter_acc} className="button b1">Shelter</button>
+          </div>
 
-        <div id="adopter_info" style={{display: 'none'}}>
-          <h1 className="title">adopter info</h1>
-          <input id="adopterName" className="input" type="text" placeholder="Name"/>
-          <input id="adopterAge" className="input top" type="text" placeholder="Age"/>
-          <input id="adopterZip" className="input top" type="text" placeholder="Location (zip code)"/>
-          <button onClick={this.acc_type} className="button b2 top" style={{marginRight: '20px'}}>Back</button>
-          <button onClick={this.welcome_adopter} className="button b2 top" style={{marginLeft: '20px'}}>Next</button>
-        </div>
+          <div id="adopter_info" style={{display: 'none'}}>
+            <h1 className="title">Adopter Info</h1>
+            <input id="email" className="input" type="email" placeholder="Email"/>
+            <input id="password" className="input top" type="password" placeholder="Password"/>
+            <input id="adopterName" className="input top" type="text" placeholder="Name"/>
+            <input id="adopterAge" className="input top" type="text" placeholder="Age"/>
```

Comment:
date of birth dropdown seems more appropriate than entering age.
for example a dropdown for each month, date, and year lines up in row.

```suggestion
            <input id="adopterAge" className="input top" type="text" placeholder="Age"/>
```